Main/src/Coffee/Action.java

```diff
@@ -0,0 +1,189 @@
+            Action.coffeeBeans -= (16 * cup);
+            Action.money += (4 * cup);
+            Action.dcups -= (1 * cup);
+            System.out.println("Price is 4$ for cup. Thank you for your purchase, please take your espresso.");
+        }
+    }
+
+    public static void latte(int cup) {
+        if (checkLatteAvailability(cup) == true) {
+            Action.water -= (350 * cup);
+            Action.milk -= (75 * cup);
+            Action.coffeeBeans -= (20 * cup);
+            Action.money += (7 * cup);
+            Action.dcups -= (1 * cup);
+            System.out.println("Price is 7$ for cup. Thank you for your purchase, please take your latte.");
+        }
+    }
+
+    public static void cappuccino(int cup) {
+        if (checkCappuccinoAvailability(cup) == true) {
+            Action.water -= (200 * cup);
+            Action.milk -= (100 * cup);
+            Action.coffeeBeans -= (12 * cup);
+            Action.money += (6 * cup);
+            Action.dcups -= cup;
+            System.out.println("Price is 6$ for cup. Thank you for your purchase, please take your cappuccino.");
+        }
+    }
+
+    public static boolean checkEspressoAvailability(int cup) {
+        float avOfWater = Action.water;
+        float avOvCoffeeBeans = Action.coffeeBeans;
+        float avOfdCups = Action.coffeeBeans;
+        System.out.println("Checking for Espresso availability...");
+        avOfWater = avOfWater / (250 * cup);
+        avOvCoffeeBeans = avOvCoffeeBeans / (16 * cup);
+        avOfdCups = avOfdCups / cup;
+        if (avOfWater >= 1 & avOvCoffeeBeans >= 1 & avOfdCups >= 1) {                // нужно округление
+            System.out.println("You can order Espresso!");
+            System.out.println("Available cups " + Math.min(Math.min(avOfWater, avOvCoffeeBeans), avOfdCups));
+            return true;
+        } else {
+            System.out.println("No enough ingredients, Please fill Machine");
+            return false;
+        }
+    }
+
+    public static boolean checkLatteAvailability(int cup) {
+        float avOfWater = Action.water;
+        float avOvCoffeeBeans = Action.coffeeBeans;
+        float avmilk = Action.milk;
+        int avOfdCups = Action.coffeeBeans;
+        System.out.println("Checking for Latte availability...");
+        avOfWater = avOfWater / (350 * cup);
+        avmilk = avmilk / (75 * cup);
+        avOvCoffeeBeans = avOvCoffeeBeans / (20 * cup);
+        avOfdCups = avOfdCups / cup;
+        if (avOfWater >= 1 & avmilk >= 1 & avOvCoffeeBeans >= 1 & avOfdCups >= 1) {
+            System.out.println("You can order Latte!");
+            System.out.println("Available cups " + Math.min(Math.min(avOfWater, avmilk), Math.min(avOvCoffeeBeans, avOfdCups)));
+            return true;
+        } else {
+            System.out.println("No enough ingredients, Please fill Machine");
+            return false;
+        }
+    }
+
+    public static boolean checkCappuccinoAvailability(int cup) {
```

Comment:
There is no need to use only static method

Reply:
fixed in Brach branch